packages/nouns-webapp/src/config.ts

```diff
@@ -152,15 +163,21 @@ const externalAddresses: Record<SupportedChains, ExternalContractAddresses> = {
   },
 };
 
+const vrgda: VrgdaConfig = {
+  firstNounId: 7975,
+  network: 'sepolia',
+  auction: '0xc27d735c529c36fc26cba27d1540429d3b8cba87',
+};
+
```

Comment:
Hardcoded this config for now. Needs to be updated here once we have mainnet contracts.